src/compiler/utilities.ts

```diff
@@ -26,20 +26,6 @@ namespace ts {
         return undefined;
     }
 
-    export function findDeclaration<T extends Declaration>(symbol: Symbol, predicate: (node: Declaration) => node is T): T | undefined;

```

Comment:
please don't remove this one. I use it in an upcoming change and would have to put it back.